src/main/java/org/worldcubeassociation/ui/ExportJSONAction.java

```diff
@@ -45,7 +46,7 @@ public void actionPerformed(ActionEvent aActionEvent) {
         }
 
         try {
-            String jsonFileName = "Results for " + getEnv().getCompetitionId() + ".json";
+            String jsonFileName = "Results (" + WorkbookAssistant.class.getPackage().getImplementationVersion() + ") for " + getEnv().getCompetitionId() + ".json";
```

Comment:
A new field in the JSON would only be useful to our purpose if the WCA website would show that info when importing (and maybe an alert on non-up-to-date WA versions).  About CCM and Cubecomps producing JSON, this problem disappears as they are web-based and thus always up to date, something that unfortunately not always happens with WA.